frontend/static_vol/js/modules/modalContents.js

```diff
@@ -54,7 +54,7 @@ const waitForOpponent = (args) => `
 `;
 
 const entryTournament = (args) => `
-    <section class="blockModal" data-modal-request_id="${args.request_id}" data-modal-username="${args.username}">
+    <section class="blockModal" data-modal-request_id="${args.request_id}" data-modal-username="${args.username}" data-modal-match_type="entryTournament">
```

Comment:
```suggestion
    <section class="blockModal" data-modal-match_type="entryTournament">
```
request_id, username使ってなさそうです